core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -556,7 +562,10 @@ public void setWaitStrategy(org.testcontainers.containers.wait.strategy.WaitStra
      * @see #waitingFor(org.testcontainers.containers.wait.strategy.WaitStrategy)
      */
     protected void waitUntilContainerStarted() {
-        getWaitStrategy().waitUntilReady(this);
+        org.testcontainers.containers.wait.strategy.WaitStrategy waitStrategy = getWaitStrategy();
```

Comment:
Is the import missing?

Reply:
no, this is the new `org.testcontainers.containers.wait.strategy.WaitStrategy` which is has a name clash with the deprecated `org.testcontainers.containers.wait.WaitStrategy`